projects/example/src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import {Component} from '@angular/core';
 import {AppRatingComponent} from './rating.component';
-import {SmartTableColumnCustomType, SmartTableConfig} from 'projects/smart-table/src/lib/smart-table/smart-table.types';
+import {SmartTableColumnCustomType, SmartTableConfig} from '@acpaas-ui-widgets/smart-table';
```

Comment:
Dit is niet de bedoeling denk ik, want je moet eventueel nieuwe zaken toch lokaal kunnen testen?